python/ray/data/read_api.py

```diff
@@ -681,7 +684,7 @@ def read_images(
     *,
     filesystem: Optional["pyarrow.fs.FileSystem"] = None,
     parallelism: int = -1,
-    meta_provider: BaseFileMetadataProvider = _ImageFileMetadataProvider(),
+    meta_provider: BaseFileMetadataProvider = DEFAULT_IMAGE_METADATA_PROVIDER,
```

Comment:
Could we set this to None by default, and then in the method body do:

```
if meta_provider is None:
    meta_provider = context.default_image_meta_provider
```

? This would provide consistency with how other default properties are configured via the context object.